src/ansys/rocky/core/launcher.py

```diff
@@ -40,6 +40,7 @@ def launch_rocky(
     *,
     headless: bool = True,
     server_port: int = DEFAULT_SERVER_PORT,
+    terminate_existing_session: bool = False,
```

Comment:
I'd suggest `close_existing`. Shorter and also `terminate` gives a sense of "killing the process", which is not the case.